lib/Raven/Client.php

```diff
@@ -881,9 +1016,12 @@ public function capture($data, $stack = null, $vars = null)
             }
 
             if (!isset($data['stacktrace']) && !isset($data['exception'])) {
-                $data['stacktrace'] = array(
-                    'frames' => Stacktrace::fromBacktrace($this, $stack, isset($stack['file']) ? $stack['file'] : __FILE__, isset($stack['line']) ? $stack['line'] : __LINE__)->getFrames(),
-                );
+                $data['stacktrace'] = [
+                    'frames' => Stacktrace::fromBacktrace(
+                        $this, $stack, isset($stack['file']) ? $stack['file'] : __FILE__,
+                        isset($stack['line']) ? $stack['line'] : __LINE__
```

Comment:
There is no need to show one particular line. The difference in one line does not make any changes when we are talking about debug. Otherwise, this is a very bad code.